recipes/jgo/meta.yaml

```
@@ -0,0 +1,40 @@
+{% set name = "jgo" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 880d880badc2b3ec88ed15f063be3fe12c83eec5c951b743b182b66d81977188
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vvv"
```

Should we install with `--no-deps` flag?
```suggestion
  script: "{{ PYTHON }} -m pip install . -vvv  --no-deps"
```

---

It's in the official "example" recipe. I think the idea is to prevent pip to download and install any packages since it should conda's job. In consequence, it's recipe's maintainers to be careful to add all the required deps in the `meta.yaml` file.

Maybe @scopatz could confirm.

---

This is fine.